exercises/concept/cars-assemble/.meta/design.md

```diff
@@ -25,3 +25,17 @@ This exercise's prerequisites Concepts are:
 
 - `basics`: know how to define methods.
 - `booleans`: know how to use boolean operators.
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer]:
+
+- `actionable`: If the student did not reuse the implementation of the `productionRatePerHour` method in the `workingItemsPerMinute` method, instruct them to do so.
+- `informative`: If the solution is hardcoding the value `221`, inform the student that it could be great to extrapolate this value to a `private final` field to make a more readable code.
+- `informative`: If the solution has `if/else-if` statements in the `productionRatePerHour` method, inform the student that it could be great to create a helper method to calculate the succes rate.
```

Comment:
```suggestion
- `informative`: If the solution has `if/else-if` statements in the `productionRatePerHour` method, inform the student that creating a helper method to calculate the succes rate might make their code easier to understand.
```

I always try to aim for an explanation of _why_ something can be done different. "it could be great" doesn't really do that IMO.